### ISSUE TITLE
no-extraneous... whatever this is _.map(x => x.foo)(vals);

### ISSUE THREAD

**Issue by graingert** [jfmengels/eslint-plugin-lodash-fp#48]
```js
#bad
 _.map(x => x.foo)(vals);

#good
_.map(x => x.foo, vals);
```

**Comment by graingert** [comment]
@jfmengels any advice on how to go about doing this?

**Comment by graingert** [comment]
or even a name for the rule

**Comment by drsjmiddleton** [comment]
Personally I think that's a shame.

Putting the additional () at the end:
(a) Makes it explicit that you're composing the function find(predicate) and then calling it with an iteratee argument
(b) The behaviour is consistent with advanced composition ie flow statements
(c) There is no advantage to the alternative  (apart from it being 1 character shorter)
